src/lib/data/creatables/bsoItems.ts

```diff
@@ -135,6 +135,141 @@ const componentRevertables: Createable[] = [
 		outputItems: {
 			[itemID('Ancestral components')]: 3
 		}
+	},
+	{
+		name: 'Revert torva full helm (broken)',
+		inputItems: {
+			[itemID('Torva full helm (broken)')]: 1
+		},
+		outputItems: {
+			[itemID('Bandosian components')]: 1
+		}
+	},
```

Comment:
```ts
brokenTorvaOutfit.map(piece => ({ name: `Revert ${itemName(piece)`, input: new Bank().add(piece), ... }))
```

Would be significantly shorter and I think easier to maintain.

Reply:
Have done it in a similar way to line 50 to keep it more consistent. Have testing it in the testing server let me know if you want me to change it like above though.